studio/schemas/fields/text.ts

```diff
@@ -53,7 +58,10 @@ export const optionalSubtitle = defineField({
   name: subtitleID.optional,
   title: "Subtitle",
   type: "string",
-  validation: (Rule) => Rule.max(60),
+  validation: (rule) => rule.max(60),
+  components: {
+    input: (props) => StringInputWithCharacterCount({ ...props, maxCount: 60 }),
+  },
 });
 
 export const richTextID = "richText";
```

Comment:
Will this field also have a character count?  
<img width="1354" alt="Screenshot 2024-09-11 at 08 48 08" src="https://github.com/user-attachments/assets/b89ab450-2124-43c3-a478-5eafc8984e19">

Reply:
Rich text fields are not supported yet. The calculation is a little more complex since the text is stored as an array of blocks instead of a regular string. I have defined a separate issue for this: #605